Investigate ffi.t fails on cpantesters

```
t/00-basic.t .. ok
# Version Info
# Mod	Lib
# 0.03	1.10
t/basic.t ..... ok
t/compile.t ... ok

# Failed test 'ffi'
# at t/ffi.t line 11.
#   libdeflate_alloc_compressor not found
#   libdeflate_deflate_compress not found
#   libdeflate_deflate_compress_bound not found
#   libdeflate_zlib_compress not found
#   libdeflate_zlib_compress_bound not found
#   libdeflate_gzip_compress not found
#   libdeflate_gzip_compress_bound not found
#   libdeflate_free_compressor not found
# Looks like you failed 1 test of 2.
t/ffi.t ....... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/2 subtests 

Test Summary Report
-------------------
t/ffi.t     (Wstat: 256 (exited 1) Tests: 2 Failed: 1)
  Failed test:  2
  Non-zero exit status: 1
Files=4, Tests=22,  4 wallclock secs ( 0.03 usr  0.04 sys +  1.36 cusr  0.49 csys =  1.92 CPU)
Result: FAIL
Failed 1/4 test programs. 1/22 subtests failed.
make: *** [Makefile:916: test_dynamic] Error 1

```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -142,5 +142,7 @@ jobs:
           echo "ALIEN_INSTALL_TYPE=${{ matrix.alien-install-type }}" >> $GITHUB_ENV
 
       - name: Run tests
+        env:
+          ALIEN_BUILD_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           cpanm --verbose --test-only dist.tar.gz

--- a/alienfile
+++ b/alienfile
@@ -22,6 +22,15 @@ plugin 'Probe::CBuilder' => (
 int main(int argc, char *argv[]) { printf("%s\n", LIBDEFLATE_VERSION_STRING); return 0; }
 END_OF_CODE
 
+meta->around_hook( gather_system => sub {
+  my $orig = shift;
+  my ($build) = @_;
+  $orig->(@_);
+  if( ! exists $build->runtime_prop->{ffi_name} ) {
+    $build->runtime_prop->{ffi_name} = 'deflate';
+  }
+});
+
 share {
   ## *bsd make is incompatible
   requires 'Alien::gmake' => 0;

